Management/src/ActuatorApi/README.md

```diff
@@ -0,0 +1,111 @@
+﻿# Steeltoe Management Sample - Actuators, Administrative Tasks, Metrics and Tracing
+
+ActuatorWeb and ActuatorApi form an ASP.NET Core-powered sample application that demonstrates how to use several Steeltoe libraries on their own and with additional tools.
+
+In order to avoid duplicating a significant amount of content, the [ActuatorWeb Readme](../ActuatorWeb/README.md) contains the shared information and this document only holds content unique to ActuatorApi.
+
+## Running locally
+
+Be sure to follow the [ActuatorWeb Readme](../ActuatorWeb/README.md) to the point where that application is running before continuing here.
+
+### MySQL
+
+This application depends on a MySQL database. Refer to [Common Tasks](../../../CommonTasks.md#MySQL) to start a server that matches the credentials defined in [appsettings.Development.json](./appsettings.Development.json).
+
+### Administrative Tasks (initialize the database)
+
+In order to demonstrate [Steeltoe Management Tasks](https://docs.steeltoe.io/api/v3/management/tasks.html), the database schema and its contents are managed as administrative tasks.
+
+1. Apply Entity Framework Core database schema migration scripts:
+
+    ```shell
+    dotnet run --runtask=MigrateDatabase
+    ```
+
+1. Run [`ForecastTask`](./AdminTasks/ForecastTask.cs) to predict weather for the next 7 days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather
+    ```
+
+    *Optional* - Add forecast data starting from a specific date and/or number of days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather --fromDate=10/18/2024 --days=30
+    ```
+
+> [!NOTE]  
+> For the `fromDate` parameter, use values formatted as `yyyy-dd-MM` or `MM/dd/yyyy`.
+
+> [!TIP]
+> The task [`ResetTask`](./AdminTasks/ResetTask.cs) can be used to remove all forecast data:
+> `dotnet run --runtask=ResetWeather`
```

Comment:
This indent is wrong now too.

Reply:
This is a limitation in GitHub markdown, documented at https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts:

> Alerts cannot be nested within other elements.